validator/accounts/v2/iface/wallet.go

```diff
@@ -0,0 +1,22 @@
+package iface
+
+import (
+	"context"
+	"io"
+)
+
+// Wallet defines a struct which has capabilities and knowledge of how
+// to read and write important accounts-related files to the filesystem.
+// Useful for keymanagers to have persistent capabilities for accounts on-disk.
+type Wallet interface {
```

Comment:
Would be nice to separate these with some comments as // Read functions or // Write functions